### ISSUE TITLE
`COMMUNITY` tab retrieval loops after 4 pages

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/YouTube-operational-API#98]
As noticed by Discord user `176659292412313600`.

The problem can be replicated with the following code:
```py
import requests, json

CHANNEL_ID = 'UC_hSjGOO67A6gIDUcyuv_8A'
pageToken = ''

while True:
    url = f'http://localhost/YouTube-operational-API/channels?part=community&id={CHANNEL_ID}'
    if pageToken != '':
        url += f'&pageToken={pageToken}'
    content = requests.get(url).text
    data = json.loads(content)
    lastItem = data['items'][-1]
    lastCommunityPostOnPage = lastItem['community'][-1]
    print(lastCommunityPostOnPage['id'], lastCommunityPostOnPage['date'])
    pageToken = lastItem['nextPageToken']
```
Output:
```
UgkxRI7nP_-y4z51b4eBXI9RF4EkOJVZwwVk 7 months ago
UgxlcI9t3lGh2flzKO14AaABCQ 1 year ago
UgzFr5ZtP5OMuEfKUld4AaABCQ 1 year ago
UgwOE_7P0L5YCEmEHh14AaABCQ 2 years ago
UgkxRI7nP_-y4z51b4eBXI9RF4EkOJVZwwVk 7 months ago
UgxlcI9t3lGh2flzKO14AaABCQ 1 year ago
UgzFr5ZtP5OMuEfKUld4AaABCQ 1 year ago
UgwOE_7P0L5YCEmEHh14AaABCQ 2 years ago
...
```

**Comment by Benjamin-Loison** [comment]
Fixed by #99.

**Comment by Benjamin-Loison** [comment]
Note that:

https://www.youtube.com/post/POST_ID

(obtained by right-clicking on `* ago` on the community post) redirects to

https://www.youtube.com/channel/CHANNEL_ID/community?lb=POST_ID
